pkg/zrouter/zrouter.go

```diff
@@ -92,10 +93,10 @@ func New(appName string, metricsServer metrics.TaskMetrics, config *Config) ZRou
 }
 
 func (r *zrouter) SetDefaultMiddlewares() {
-	r.Use(zmiddlewares.ErrorHandlerMiddleware(r.config.Logger))
+	r.Use(zmiddlewares.ErrorHandlerMiddleware())
 	r.Use(zmiddlewares.RequestID())
```

Comment:
Shouldn't we register the RequestID middleware first? So the error middleware will log the error with the id too

Reply:
Nice catch!